s3-cloudfront-spa/template.tf

```diff
@@ -0,0 +1,145 @@
+terraform {
+    backend "s3" {
+      key = "terraform.tfstate"
+      region = "eu-west-2"
```

Comment:
Could this be made configurable easily? Or at least call it out in the deployment instructions

Reply:
Yes i will make it configurable by adding variables for bankend key, region and bucket

Reply:
I will remove the terraform remote setup